packages/server/src/dataset/dataset.entity.ts

```diff
@@ -84,6 +84,10 @@ export class Dataset {
 
   @Field(type => [ScoredDataset])
   thematicallySimilarDatasets: Promise<Dataset[]>;
+
+  @Field({ nullable: false })
```

Comment:
Please add a block comment above this line documenting this field. `isTest` is definitely an idiosyncratic field we're having to keep track of now so we should definitely document why this field exists and what it's used for for future volunteers